GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SceneObjectActivity.java

```diff
@@ -76,6 +73,34 @@ private boolean checkCameraHardware(Context context) {
                 PackageManager.FEATURE_CAMERA);
     }
 
+    private long prevTime = 0;
+    private int numFrames = 0;
+    private PreviewCallback previewCallback = new PreviewCallback() {
+
+        @Override
+        /**
+         * The byte data comes from the android camera in the yuv format. so we
+         * need to convert it to rgba format.
+         */
+        public void onPreviewFrame(byte[] data, Camera camera) {
+            numFrames++;
+            if (numFrames == 100) {
+                Parameters params = SceneObjectActivity.this.camera
```

Comment:
that was part of my attempt to prevent the camera from auto-focusing. but apparently it seems that it is not needed because even without this code, the camera doesn't auto-focus in 60 fps mode. So I will remove this code.